src/Myg-MineSweeper/MineSweeper.class.st

```diff
@@ -44,9 +44,8 @@ MineSweeper class >> menuHelp [
 
 	| menu |
 	menu := ToMenu new.
-	menu label: (ToLabel text: 'Help').
-	menu icon:
-		(ToImage inner: (Smalltalk ui icons iconNamed: #glamorousHelp)).
+	menu labelText: 'Help'.
+	menu iconImage: (Smalltalk ui icons iconNamed: #glamorousHelp).
```

Comment:
We should check because we should not continue to use this Smalltalk ui icons everywhere and in addition we should use #help and not #glamorousHelp - if there is no help we should introduce it.

Reply:
Okay, here I only did refactoring by using some convenient API in Toplo to compact the Myg code and improve readability (may be not necessary).

The problem of this PR is that I mixed:
- Upgrading Toplo to ~ master by end of August, but not based on a Toplo's fixed version
- Fix initial size some window was being opened with too high resolution to my screen (I think 1000@1000)
- These kind of random refactorings that were an excuse to explore and learn of Myg

Reply:
The icons in this case, I would like to read PNGs from game resources. This not only resolves the dependency on this Smalltalk API, but also to use PNGs with higher resolution.

Reply:
Created https://github.com/Ducasse/Myg/issues/6 with this proposal